2021-02-02 09:51:57.055029: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-02 09:52:02.244104: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2021-02-02 09:52:02.302713: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.303771: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-02 09:52:02.303807: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-02 09:52:02.306132: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-02 09:52:02.307954: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-02 09:52:02.308341: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-02 09:52:02.310645: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-02 09:52:02.312017: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-02 09:52:02.317228: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-02 09:52:02.317381: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.318489: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.319455: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
Using TensorFlow version:  2.3.0 , GPU: 1
Using Keras version:  2.4.0
2021-02-02 09:52:02.395019: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2500115000 Hz
2021-02-02 09:52:02.396262: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5561720 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-02 09:52:02.396291: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-02 09:52:02.661380: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.662370: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55cdb90 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2021-02-02 09:52:02.662405: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 Ti, Compute Capability 7.5
2021-02-02 09:52:02.662634: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.665105: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-02 09:52:02.665153: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-02 09:52:02.665189: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-02 09:52:02.665227: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-02 09:52:02.665254: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-02 09:52:02.665280: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-02 09:52:02.665305: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-02 09:52:02.665331: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-02 09:52:02.665426: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.666603: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:02.667699: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2021-02-02 09:52:02.667746: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-02 09:52:04.394492: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-02-02 09:52:04.394555: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2021-02-02 09:52:04.394573: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2021-02-02 09:52:04.394799: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:04.397728: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-02 09:52:04.398589: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10068 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:04:00.0, compute capability: 7.5)
The inputs are normalized to have a unit Gaussian distribution
Model: "CNN-POD"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
high-res-input (InputLayer)  [(None, 3, 192, 192)]     0         
_________________________________________________________________
predic_01 (Conv2D)           (None, 128, 192, 192)     9728      
_________________________________________________________________
predic_02 (BatchNormalizatio (None, 128, 192, 192)     512       
_________________________________________________________________
predic_03 (MaxPooling2D)     (None, 128, 96, 96)       0         
_________________________________________________________________
predic_04 (Conv2D)           (None, 256, 96, 96)       295168    
_________________________________________________________________
predic_05 (BatchNormalizatio (None, 256, 96, 96)       1024      
_________________________________________________________________
predic_06 (MaxPooling2D)     (None, 256, 48, 48)       0         
_________________________________________________________________
predic_07 (Conv2D)           (None, 256, 48, 48)       590080    
_________________________________________________________________
predic_08 (BatchNormalizatio (None, 256, 48, 48)       1024      
_________________________________________________________________
predic_09 (MaxPooling2D)     (None, 256, 24, 24)       0         
_________________________________________________________________
predic_10 (Conv2D)           (None, 512, 24, 24)       1180160   
_________________________________________________________________
predic_11 (BatchNormalizatio (None, 512, 24, 24)       2048      
_________________________________________________________________
predic_12 (MaxPooling2D)     (None, 512, 12, 12)       0         
_________________________________________________________________
predic_13 (Conv2D)           (None, 512, 12, 12)       2359808   
_________________________________________________________________
predic_14 (BatchNormalizatio (None, 512, 12, 12)       2048      
_________________________________________________________________
predic_15 (Conv2D)           (None, 64, 12, 12)        294976    
=================================================================
Total params: 4,736,576
Trainable params: 4,733,248
Non-trainable params: 3,328
_________________________________________________________________
None
2021-02-02 09:52:22.060447: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 2981 of 4200
2021-02-02 09:52:26.464448: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
2021-02-02 09:52:28.248164: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-02 09:52:33.445091: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-02 09:56:23.325007: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 1969 of 4200
2021-02-02 09:56:33.313612: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 4047 of 4200
2021-02-02 09:56:34.020224: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
Epoch 0001/0030, loss: 6.685762405395508, val_loss: 2.223954439163208, elapsed time from start: 301.4956967830658
2021-02-02 10:01:12.914626: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 2547 of 4200
2021-02-02 10:01:19.131782: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
Epoch 0002/0030, loss: 0.9732789993286133, val_loss: 0.3176438510417938, elapsed time from start: 583.872542142868
Epoch 0003/0030, loss: 0.14504452049732208, val_loss: 0.05443933233618736, elapsed time from start: 837.3223533630371
Epoch 0004/0030, loss: 0.029607541859149933, val_loss: 0.01689070835709572, elapsed time from start: 1094.4283447265625
Epoch 0005/0030, loss: 0.01121285930275917, val_loss: 0.009378474205732346, elapsed time from start: 1348.9370698928833
Epoch 0006/0030, loss: 0.008327440358698368, val_loss: 0.008497770875692368, elapsed time from start: 1606.1219856739044
Epoch 0007/0030, loss: 0.007801945321261883, val_loss: 0.008302589878439903, elapsed time from start: 1864.7358651161194
Epoch 0008/0030, loss: 0.007594708818942308, val_loss: 0.00857260450720787, elapsed time from start: 2120.0112047195435
Epoch 0009/0030, loss: 0.007466096896678209, val_loss: 0.007704535964876413, elapsed time from start: 2373.930834054947
Epoch 0010/0030, loss: 0.007373410742729902, val_loss: 0.007722180336713791, elapsed time from start: 2627.2846217155457
Epoch 0011/0030, loss: 0.00729756336659193, val_loss: 0.007545568980276585, elapsed time from start: 2880.8522284030914
Epoch 0012/0030, loss: 0.007191014010459185, val_loss: 0.007588363252580166, elapsed time from start: 3134.414664745331
Epoch 0013/0030, loss: 0.007101354189217091, val_loss: 0.007733497768640518, elapsed time from start: 3388.533348083496
Epoch 0014/0030, loss: 0.007027802988886833, val_loss: 0.007478999439626932, elapsed time from start: 3642.723724126816
Epoch 0015/0030, loss: 0.00696171447634697, val_loss: 0.007089673075824976, elapsed time from start: 3896.995807647705
Epoch 0016/0030, loss: 0.006903695408254862, val_loss: 0.0070841871201992035, elapsed time from start: 4150.967001914978
Epoch 0017/0030, loss: 0.006856469437479973, val_loss: 0.007072691805660725, elapsed time from start: 4405.634578704834
Epoch 0018/0030, loss: 0.006811398547142744, val_loss: 0.0069708023220300674, elapsed time from start: 4661.909733772278
Epoch 0019/0030, loss: 0.006770385894924402, val_loss: 0.006871430203318596, elapsed time from start: 4920.135450124741
Epoch 0020/0030, loss: 0.00673659797757864, val_loss: 0.006871591322124004, elapsed time from start: 5176.280802965164
Epoch 0021/0030, loss: 0.006703366991132498, val_loss: 0.006830528844147921, elapsed time from start: 5432.380393743515
Epoch 0022/0030, loss: 0.006674951873719692, val_loss: 0.006791573483496904, elapsed time from start: 5691.426419019699
Epoch 0023/0030, loss: 0.006647863890975714, val_loss: 0.006735780276358128, elapsed time from start: 5949.422812461853
Epoch 0024/0030, loss: 0.006624236237257719, val_loss: 0.006693064235150814, elapsed time from start: 6205.0000603199005
Epoch 0025/0030, loss: 0.006602701265364885, val_loss: 0.006936138495802879, elapsed time from start: 6462.158648729324
Epoch 0026/0030, loss: 0.006583770737051964, val_loss: 0.006662321276962757, elapsed time from start: 6719.283148288727
Epoch 0027/0030, loss: 0.006566744763404131, val_loss: 0.006648411974310875, elapsed time from start: 6979.270637273788
Epoch 0028/0030, loss: 0.006548936944454908, val_loss: 0.006623119115829468, elapsed time from start: 7238.228357553482
Epoch 0029/0030, loss: 0.006534620188176632, val_loss: 0.0066124554723501205, elapsed time from start: 7494.909870862961
Epoch 0030/0030, loss: 0.006522868759930134, val_loss: 0.006601990666240454, elapsed time from start: 7753.907890796661
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2021-02-02 12:01:37.400655: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
